relayer/app/state.go

```diff
@@ -43,16 +43,16 @@ func (s *State) Clear(dstID uint64) error {
 	return s.saveUnsafe()
 }
 
-// Persist saves the given height for the given chainID.
-func (s *State) Persist(dstID, srcID, height uint64) error {
+// Persist saves the given offset for the given src and dest chain pair..
+func (s *State) Persist(dstID, srcID, offset uint64) error {
 	s.mu.Lock()
 	defer s.mu.Unlock()
 
 	srcMap, ok := s.cursors[dstID]
 	if !ok {
 		srcMap = make(map[uint64]uint64)
 	}
-	srcMap[srcID] = height
+	srcMap[srcID] = offset
```

Comment:
How do we map offest to source block height (which is we're we need to start looking) - ask halo?

Reply:
yeah, we query `latestAttestation` which has a `height` and `offset`, we start streaming from that point.